src/System Application/App/Telemetry/src/Feature Telemetry/FeatureTelemetryImpl.Codeunit.al

```diff
@@ -42,6 +42,19 @@ codeunit 8704 "Feature Telemetry Impl."
         LogMessage(EventId, EventName, Verbosity::Normal, CallerCustomDimensions, UsageCustomDimensions, CallerModuleInfo, CallerCallStackModuleInfos);
     end;
 
+    procedure LogWarning(EventId: Text; FeatureName: Text; EventName: Text; WarningText: Text; CallerCustomDimensions: Dictionary of [Text, Text]; CallerModuleInfo: ModuleInfo)
```

Comment:
afair FeatureTelemetry is meant to be used for statistics... like, how much uptake do we have, how much usage do we have... I think the error bit was added to be able to see painpoints... 

For other things we have the Telemetry codeunit (same module here) and that is used for logging all sorts of telemetry, i.e. the normal stuff...

If we're adding warnings to feature telemetry, is there work to update the PowerBI dashboards etc? because if not, then I think you should use Telemetry and not FeatureTelemetry...